tests/common/bakery_chain/lineitem.rs

```diff
@@ -1,11 +1,12 @@
+//! SeaORM Entity. Generated by sea-orm-codegen 0.2.0
+
 use sea_orm::entity::prelude::*;
 
 #[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
 #[sea_orm(table_name = "lineitem")]
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: i32,
-    #[sea_orm(column_type = "Decimal(Some((19, 4)))")]
```

Comment:
Is it possible to get the required info from SeaSchema for this?

Reply:
Fixed on 39c5a4d. But I'm thinking what `column_type` should also be generated by codegen, perhaps `Text`?